### ISSUE TITLE
Upgrade failed  with 6 "result: false" 

### ISSUE THREAD

**Issue by MnFrozenFish** [REMnux/remnux-cli#135]
I created a REMnux VM on VMWare Workstation Pro V17 following the link below: https://docs.remnux.org/install-distro/get-virtual-appliance

While I was working on step 6 (upgrade the REMnux VM),  the upgrade went on well for about 30 minutes and then errored out.  I tried the upgrade 2 more times after restarted the VM, and I also refreshed APT package database, but the upgrade attempts all failed.  Below is the snippet of the saltstack.log that contains all the errors (grep -B8 'result: false'  _logfile_):

![image](https://github.com/REMnux/remnux-cli/assets/59742166/e306b981-5c78-4cda-8831-e4adabc45a5f)

**Comment by digitalsleuth** [comment]
Hi @MnFrozenFish , can you provide us with your saltstack.log file so we can see what might have occurred?

**Comment by MnFrozenFish** [comment]
Sure. Herewith the log
[saltstack.log](https://github.com/REMnux/remnux-cli/files/11707875/saltstack.log)

**Comment by digitalsleuth** [comment]
Hi @MnFrozenFish , it looks like there was an SSL error when attempting to download SSView (as you indicated with your screenshot):
```bash
# [DEBUG   ] Requesting URL https://www.mitec.cz/Downloads/SSView.zip using GET method
# [DEBUG   ] Using backend: tornado
# [WARNING ] SSL Error on 11 ('146.112.250.194', 443): [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)
# [DEBUG   ] Cannot perform 'http.query': https://www.mitec.cz/Downloads/SSView.zip - [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)
# [ERROR   ] Failed to cache https://www.mitec.cz/Downloads/SSView.zip: Error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131) reading /Downloads/SSView.zip
```
Everything else in the installation completed successfully though. The SSL error can sometimes occur when a certificate on the page has an error, or the date/time on your system is either before or after the certificate validity window. Once you confirm your date/time, and are able to successfully visit the URL manually, you should be fine to then try again if you wish.

**Comment by MnFrozenFish** [comment]
Thanks. I will check and let you know how it goes.

**Comment by MnFrozenFish** [comment]
Herewith some feedback: 
1.  I ran the upgrade again this morning and the above certificate-verify error is still there. 
2. The date/time on my REMnux VM is correct and it is within the validity window (see attached the cert snapshot)
![image](https://github.com/REMnux/remnux-cli/assets/59742166/e0d210ec-fdb0-4e78-a620-85c2670f324e)
3. I was able to download the SSView.zip file by putting the URL into Firefox. Interestingly, Firefox sometimes will pop up a security warning screen and does not download, sometimes it just downloads the file without problem 

What should I do from here now that I have the SSView.zip file?
